libloki/api.js

```diff
@@ -14,50 +14,43 @@
     SESSION_BACKGROUND_MESSAGE: 32,
     GROUP_REQUEST_INFO: 64,
     NORMAL_FRIEND_REQUEST_MESSAGES: 128,
-    // If you add anything, be sure it is bitwise safe! (unique and 2 multiples)
+    // If you add any new flag, be sure it is bitwise safe! (unique and 2 multiples)
+    ALL: 65535,
   };
 
-  const debugFlags =
-    DebugFlagsEnum.GROUP_SYNC_MESSAGES |
-    DebugFlagsEnum.CONTACT_SYNC_MESSAGES |
-    DebugFlagsEnum.AUTO_FRIEND_REQUEST_MESSAGES |
-    DebugFlagsEnum.SESSION_REQUEST_MESSAGES |
-    DebugFlagsEnum.SESSION_MESSAGE_SENDING |
-    DebugFlagsEnum.SESSION_BACKGROUND_MESSAGE |
-    DebugFlagsEnum.NORMAL_FRIEND_REQUEST_MESSAGES |
-    DebugFlagsEnum.GROUP_REQUEST_INFO;
+  const debugFlags = DebugFlagsEnum.ALL;
 
   // feel free to change this to window.console.warn to have a stack with all those logs
   // disable loki logs
   const debugLogFn = undefined; // window.console.warn
```

Comment:
Could we just make this an empty function? This way you don't have to check to see if it exists